### ISSUE TITLE
model is not creating

### ISSUE THREAD

**Issue by obulrdy6881** [alfredfrancis/ai-chatbot-framework#135]
Failed to load nlu model. Maybe path G:\AI_chatbot\ai-chatbot-framework\model_files doesn't exist
![Screenshot (58)](https://user-images.githubusercontent.com/62610390/81987782-fd0c8a00-9657-11ea-8020-bde4e9e2120b.png)

**Comment by surya-vamsi2310** [comment]
@alfredfrancis  when we are tring to do with out docker we are getting this... can you please check

**Comment by LoveMeWithoutAll** [comment]
If there is not model_files, manage.py would make models.
My terminal log is below

(ai-chatbot-framework) C:\workspace\dev-study\ai-chatbot-framework>python manage.py migrate
C:\ProgramData\Anaconda3\envs\ai-chatbot-framework\lib\site-packages\h5py\__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `
np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
--------------------------------------------------------------------------------
WARNING in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:587]:
Failed to load nlu model. Maybe path C:\workspace\dev-study\ai-chatbot-framework\model_files doesn't exist
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in utils [C:\workspace\dev-study\ai-chatbot-framework\app\endpoint\utils.py:25]:
loaded synonyms {}
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in controllers [C:\workspace\dev-study\ai-chatbot-framework\app\endpoint\controllers.py:224]:
Intent Model updated
--------------------------------------------------------------------------------
Created default bot
Imported 3 Stories
Training models..
--------------------------------------------------------------------------------
DEBUG in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:479]:
Check if num_neg 10 is smaller than number of intents 2, else set num_neg to the number of intents - 1
--------------------------------------------------------------------------------
WARNING:tensorflow:From C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:485: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:492: The name tf.placeholder_with_default is deprecated. Please use tf.compat.v1.placeholder_with_default instead
.

WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

WARNING:tensorflow:From C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:272: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.Dense instead.
WARNING:tensorflow:From C:\ProgramData\Anaconda3\envs\ai-chatbot-framework\lib\site-packages\tensorflow_core\python\layers\core.py:187: Layer.apply (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed i
n a future version.
Instructions for updating:
Please use `layer.__call__` method instead.
WARNING:tensorflow:From C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:273: dropout (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dropout instead.
WARNING:tensorflow:From C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:323: The name tf.losses.get_regularization_loss is deprecated. Please use tf.compat.v1.losses.get_regularization_
loss instead.

WARNING:tensorflow:From C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:497: The name tf.train.AdamOptimizer is deprecated. Please use tf.compat.v1.train.AdamOptimizer instead.

WARNING:tensorflow:From C:\ProgramData\Anaconda3\envs\ai-chatbot-framework\lib\site-packages\tensorflow_core\python\ops\math_grad.py:1424: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future ve
rsion.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:500: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

2020-11-18 13:13:53.740784: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
WARNING:tensorflow:From C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:368: The name tf.global_variables_initializer is deprecated. Please use tf.compat.v1.global_variables_initializer
 instead.

--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 10 / 300: loss 0.6887957453727722, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 20 / 300: loss 0.581013023853302, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 30 / 300: loss 0.5026983022689819, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 40 / 300: loss 0.4220612347126007, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 50 / 300: loss 0.3563912510871887, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 60 / 300: loss 0.3342028260231018, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 70 / 300: loss 0.31353944540023804, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 80 / 300: loss 0.2943435609340668, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 90 / 300: loss 0.27651268243789673, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 100 / 300: loss 0.2599477469921112, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 110 / 300: loss 0.24456223845481873, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 120 / 300: loss 0.2302355021238327, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 130 / 300: loss 0.21688321232795715, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 140 / 300: loss 0.20443032681941986, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 150 / 300: loss 0.19282950460910797, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 160 / 300: loss 0.1819973737001419, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 170 / 300: loss 0.17186328768730164, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 180 / 300: loss 0.16238239407539368, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 190 / 300: loss 0.15351130068302155, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 200 / 300: loss 0.14637985825538635, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 210 / 300: loss 0.137509286403656, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 220 / 300: loss 0.13028499484062195, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 230 / 300: loss 0.12371155619621277, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 240 / 300: loss 0.11744002252817154, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 250 / 300: loss 0.11147581785917282, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 260 / 300: loss 0.1058787927031517, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 270 / 300: loss 0.10070260614156723, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 280 / 300: loss 0.0958031639456749, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 290 / 300: loss 0.0911443829536438, train accuracy : 1.000
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in starspace_intent_classifier [C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:410]:
epoch 300 / 300: loss 0.08674509078264236, train accuracy : 1.000
--------------------------------------------------------------------------------
WARNING:tensorflow:From C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:666: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.

WARNING:tensorflow:From C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:593: The name tf.train.import_meta_graph is deprecated. Please use tf.compat.v1.train.import_meta_graph instead.

WARNING:tensorflow:From C:\workspace\dev-study\ai-chatbot-framework\app\nlu\classifiers\starspace_intent_classifier.py:597: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

--------------------------------------------------------------------------------
INFO in utils [C:\workspace\dev-study\ai-chatbot-framework\app\endpoint\utils.py:25]:
loaded synonyms {}
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
INFO in controllers [C:\workspace\dev-study\ai-chatbot-framework\app\endpoint\controllers.py:224]:
Intent Model updated
--------------------------------------------------------------------------------
Training models finished..
